CONTRIBUTING.md

```diff
@@ -1,13 +1,60 @@
-# Web Platform Incubator Community Group
+# Contributing Guide
 
-This repository is being used for work in the W3C Web Platform Incubator Community Group, governed by the [W3C Community License
-Agreement (CLA)](http://www.w3.org/community/about/agreements/cla/). To make substantive contributions,
-you must join the CG.
+You are welcome to contribute to this specification. In this guide, we lay out
+the processes we try to stick to as we work on it.
 
-If you are not the sole contributor to a contribution (pull request), please identify all
-contributors in the pull request comment.
+## Filing an Issue
 
-To add a contributor (other than yourself, that's automatic), mark them one per line as follows:
+If you're reading the specification and something is confusing, broken, has a typo,
+or you feel like important use cases or information are missing, please let us
+know by filing an issue.
+
+A few useful tips:
+
+* It's good idea to check out the
+[list of other open issues](https://github.com/WICG/ua-client-hints/issues) in
+case one already exists. Comments to existing issues are a great way to
+contribute.
+* Choose a descriptive title when creating a new issue
+
+## Pull Requests
+
+Pull requests to this specification are encouraged and welcome.
+
+Before doing the work to update the spec, please start with a new issue so we
+can ensure that your changes align with the goals of this specification (or so
+you can explain why the goals of this specification should change, if that's the
+case).
+
+Please note the next section about CLA requirements before any PR can be
```

Comment:
The CLA section is no longer next ;)

Reply:
Heh, good catch. I promise it was when I wrote that!